app/src/pages/inside/projectSettingsPageContainer/content/defectTypes/modals/addEditDefectTypeModal/addEditDefectTypeModal.jsx

```diff
@@ -110,14 +110,18 @@ const AddEditDefectTypeModal = ({
   data: { onSave, defectType, defectTypes },
   handleSubmit,
   initialize,
-  dirty,
   change,
+  currentValue,
+  initialValue,
 }) => {
   const { formatMessage } = useIntl();
   const dispatch = useDispatch();
 
   const [color, setColor] = useState();
```

Comment:
My suggestion is to rewrite this line to `  const [color, setColor] = useState(defectType.color);`